specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManager.json

```diff
@@ -0,0 +1,682 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Put Network Manager": {
+            "$ref": "./examples/NetworkManagerPut.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "NetworkManagers_Delete",
+        "description": "Deletes a network manager.",
+        "responses": {
+          "200": {
+            "description": "Delete Succeed."
+          },
+          "204": {
+            "description": "Request successful. The resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkManagersDelete": {
+            "$ref": "./examples/NetworkManagerDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/commit": {
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        }
+      ],
+      "post": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "NetworkManagerCommits_Post",
+        "description": "Post a Network Manager Commit.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerCommit"
+            },
+            "description": "Parameters supplied to specify which Managed Network commit is."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Commit Succeed"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkManageCommitPost": {
+            "$ref": "./examples/NetworkManagerCommitPost.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/deploymentStatus": {
```

Comment:
consider renaming 'deploymentStatus' to 'listDeploymentStatus'

Reply:
Using list* actions can enable the usage of these in template deployments.
Refer https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource?tabs=json#list